README.md

```diff
@@ -13,36 +13,50 @@ Previously, these contracts were part of `v2-periphery`. You can find audit repo
 ### v1.4.0
 
 [ad350f2]: https://github.com/sablier-labs/airdrops/tree/ad350f2
+[d4eb8de]: https://github.com/sablier-labs/airdrops/tree/d4eb8de
 
-| Auditors | Types | Commits              | Reports                                              |                                                                                |
-| :------- | :---- | :------------------- | :--------------------------------------------------- | ------------------------------------------------------------------------------ |
-| Cantina  | Firm  | [`ad350f2`][ad350f2] | [2025-04-05](./airdrops/v1.4.0/20250405_cantina.pdf) | ![image](https://files.sablier.com/external/badges/cantina-airdrops-1-3-0.svg) |
+| Auditors | Types | Commits              | Reports                                              |                                                                                                                                       |
+| :------- | :---- | :------------------- | :--------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| Cantina  | Firm  | [`d4eb8de`][d4eb8de] | [2025-07-28](./airdrops/v1.4.0/20250728_cantina.pdf) | [![image](https://files.sablier.com/external/badges/cantina.svg)](https://cantina.xyz/portfolio/6ace524c-aa43-42ec-afcc-ea9173ba4679) |
```

Comment:
Let's define the Cantina image separately:

```markdown
[cantina-img]: https://files.sablier.com/external/badges/cantina.svg
```

And then, reference it like so (note, double-check the syntax, not sure this is the one):

```markdown
[![cantina][cantina-img]]
```

Reply:
Nice catch and good feedback.

Reply:
https://github.com/sablier-labs/audits/pull/18/commits/28b6d5ac2249753c93bb887d377ab4566430f146